presto-hive/src/test/java/io/prestosql/plugin/hive/TestOriginalFilesUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive;
+
+import com.google.common.io.Resources;
+import io.prestosql.orc.OrcReaderOptions;
+import io.prestosql.plugin.hive.orc.OriginalFilesUtils;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.mapred.JobConf;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import static io.prestosql.plugin.hive.AcidInfo.OriginalFileInfo;
+import static io.prestosql.plugin.hive.HiveTestUtils.HDFS_ENVIRONMENT;
+import static io.prestosql.testing.TestingConnectorSession.SESSION;
+import static org.testng.Assert.assertEquals;
+
+public class TestOriginalFilesUtils
+{
+    private String tablePath;
+    private Configuration config;
+
+    @BeforeClass
+    public void setup()
+            throws Exception
+    {
+        tablePath = new File(Resources.getResource(("dummy_id_data_orc")).toURI()).getPath();
```

Comment:
It would be nicer to generate data files from test code

Reply:
These files are generated by Hive, in UTs we don't have the provision to start Hive and generate files. Anyway, these are small files, shouldn't be an issue to keep, as we already keep such files for TestOrcDeletedRows too.

Reply:
Indeed, we could use our own ORC writer for them. @losipiuk issue?

Reply:
https://github.com/prestosql/presto/issues/4555